### PR TITLE
Update workflows for dev and test to include Cypress tests

### PR DESCRIPTION
**Pull request by cshnimble**
Introduces Cypress tests to the GitHub actions for deployments to development and test environments.

Will require (if they don't already exist) some secrets adding to GitHub. Alternatively, if they already exist and the command needs updating:

- [ ] URL: The url of the academisation api in dev and test
- [ ] API_KEY: The api key for the academisation api in dev and test

### CODE REVIEW

File: .github/workflows/build-and-push-image-development.yml [hunk 1]
```diff
@@ -59,3 +59,26 @@ jobs:
...
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+      - name: Npm install
+        run: npm install
+      - name: Run cypress
+        run: npm run cy:run
```
Comment (cshnimble) [inline .github/workflows/build-and-push-image-development.yml:79]: @dangood84 Are there any command line flags we need to be passing to cypress here? e.g. api keys

**Comment by DrizzlyOwl** [comment]
Hi @cshnimble great work here. This repo is using an older deployment workflow so I would like to propose we update it to a more up-to-date version which combines the individual YML files into 1 single workflow. You can see how we have [incorporated the Cypress tests in other repos here](https://github.com/DFE-Digital/trams-data-api/blob/main/.github/workflows/build-and-push-image.yml#L135-L166) so I would prefer to update the whole workflow and bundle your Cypress test into that change.

What do you think?

**Comment by cshnimble** [comment]
Hi @DrizzlyOwl I'm totally on board with that, can either do it as part of this PR or as a separate one and close this if you prefer?

**Review by DrizzlyOwl — APPROVED** [review]
Secrets have been updated to include the ones referenced in the PR

**Pull request merged**